src/modules/template-management/template-form/TemplateForm.tsx

```diff
@@ -64,15 +55,13 @@ const TemplateForm = () => {
     setActiveStep((prevActiveStep) => prevActiveStep - 1);
   }, []);
 
-  const decoratedSteps = useMemo(() => decorateSteps(formManagers), [formManagers]);
-
   return pending ? (
-    <CircularProgress className={csx.loader} />
+    <Loader />
   ) : (
     <>
       <StepHeader description={descriptions[activeStep]} label={steps[activeStep].label} />
 
```

Comment:
Ok I can see that descriptions are being used here according to the step number. I think I get it now.

Reply:
Y descriptions are needed only for this StepHeader component to putting them inside same object will be mistake and generates a little bit more code